foundations/javascript_basics/fundamentals-5.md

```diff
@@ -50,4 +52,4 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-*   It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
\ No newline at end of file
+*   It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
```

Comment:
> I don't know why does it show as changed if I didn't change anything about that

Yeah, this sometimes show up when there's no newline at the end of the file. So that it does not occur in the future, we can fix this by just adding an empty line at the end of the file. Feel free to make this change if you can!

Reply:
Sure! Thanks for the explanation.

Reply:
It looks like it already had a newline. Strange.

Reply:
By a newline, I mean an empty line after line 55. Currently, line 55 is the last line of the file, we should have an empty line at the end of the file i.e line 56. I've attached a screenshot below where it can be seen that line 55 is the last.


![image](https://user-images.githubusercontent.com/85733202/168628587-c60d8f24-64b3-486f-b924-0ec6326ff438.png)

Reply:
Strange. We don't have a newline at the end of this file. And I can't see the newline on your branch as well. You can run `git log -p` command to check if you actually committed and pushed a commit with a newline. Try that, if it works, then great. Otherwise, I can add a newline myself and merge this pull request.

Thanks for bearing with me.